src/test/regress/sql/multi_extension.sql

```diff
@@ -112,6 +112,9 @@ ALTER EXTENSION citus UPDATE TO '9.0-1';
 ALTER EXTENSION citus UPDATE TO '9.0-2';
 ALTER EXTENSION citus UPDATE TO '9.1-1';
 ALTER EXTENSION citus UPDATE TO '9.2-1';
+ALTER EXTENSION citus UPDATE TO '9.2-2';
+ALTER EXTENSION citus UPDATE TO '9.3-1';
```

Comment:
I think adding a comment before the `9.3-1` upgrade will help a lot in explaining why we have an upgrade in there that doesn't have an upgrade path to it.